client/lib/pages/main_pages/app_tab_router.dart

```diff
@@ -7,7 +7,7 @@ import 'package:who_app/api/stats_store.dart';
 import 'package:who_app/api/user_preferences_store.dart';
 import 'package:who_app/api/who_service.dart';
 import 'package:who_app/constants.dart';
-import 'package:who_app/pages/main_pages/check_up_poster_page.dart';
+import 'package:who_app/pages/main_pages/check_up_intro_page.dart';
```

Comment:
Please add the `import ....check_up_poster_page.dart';` as some versions of the app will use the poster where we can't get the check-up tool approved. So the client should import both packages and then allow toggling through the yaml file.

Can you test that switching the yaml changes what's displayed? Then confirm that in a PR comment.

Reply:
I'm adding a controls section to the content bundles and using it in the home_index.yaml like this: ```controls:
    - show_symptom_checker: true```